table-paste-col/table-paste-col.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import sys
+import argparse 
+
+"""
+Add a column containing a constant value to a table. Header assumed. 
+"""
+def get_input_file_object(filename):
+    if filename == "-":
+        return sys.stdin
+    return open(filename, "r")
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("-t", "--table", default='-', help="input csv file; STDIN if - (default: %(default)s")
+parser.add_argument("-c", "--colname", help="name for new column")
+parser.add_argument("-v", "--colvalue", help="constant value for new column")
+parser.add_argument("-s", "--separator", help="column separator character. use $ as prefix")
```

Comment:
--sep

$ as prefix is only used for tabs. Other separators don't need this eg ",". I would remove this note from here as it is a Bash thing, not a concern of this script. 

use a default by adding `, default="\t"`